include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_histogram.h

```diff
@@ -0,0 +1,681 @@
+// -*- C++ -*-
+//===-- parallel_backend_sycl_reduce.h --------------------------------===//
```

Comment:
This file name comment should be `parallel_backend_sycl_histogram.h`

Reply:
good catch, fixed.